### ISSUE TITLE
heron/config/  cleanup 

### ISSUE THREAD

**Issue by huijunwu** [twitter/heron#1777]
Do we use this directory any more https://github.com/twitter/heron/tree/master/config ?
If we don't use it, we may remove/cleanup it

**Comment by kramasamy** [comment]
Yes, it is being used for config generation. When we move to 0.4.4 or 0.5.0 of bazel, we can look at removing them.
